syntax = "proto3";

package protos;

message InputReader {
  bool shuffle = 1;
  oneof input_reader{
    ImageInputReader image_input_reader = 2;
  }
  uint32 num_parallel_batches = 3;
  int32 num_prefetch_batches = 8;
}

message ImageInputReader {
  string input_path = 1;
}